authors/apps/articles/views.py

```diff
@@ -57,3 +61,72 @@ def destroy(self, request, pk, **kwargs):
                 return Response("You have succesfully deleted the article")
             return Response("You do not have permision to delete the article")
         return Response(self.err_message, status=status.HTTP_404_NOT_FOUND)
+
+
+class Like(APIView):
+    permission_classes = (IsAuthenticated, )
+
+    @classmethod
+    def post(self, request, **kwargs):
+        current_user = request.user.username
+        article = Article.objects.filter(id=kwargs['article_id'])
+        if article:
```

Comment:
@KabohaJeanMark So if you make the suggested implementations this `if block` is unnecessary.

Reply:
Yes, you are right @archibishop